class.phpmailer.php

```diff
@@ -1364,20 +1364,24 @@ public function postSend()
      */
     protected function sendmailSend($header, $body)
     {
-        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped by escapeshellcmd when popen is called due to safe mode.
-        if (!empty($this->Sender) || ini_get('safe_mode')) {
+        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped.
```

Comment:
Probably not.  I was trying to fix the CVEs without giving away too much about the underlying issues, hence the incompleteness, but I've pretty much given up on that with this latest pull request.  However, it's 7 AM here and I haven't slept, so it's certainly possible I'm missing something.